### ISSUE TITLE
false positive for CVE-2019-16374 for npm platform

### ISSUE THREAD

**Issue by acthtml** [anchore/grype#817]
**What happened**:
grype results in CVE-2019-16374 for [npm package platform](https://www.npmjs.com/package/platform).  However, npm platform.js is a helper tool which returns os version, name..., it is nothing refers to  CVE-2019-16374(Pega Platform)

**What you expected to happen**:
No false positive for CVE-2019-16374 when using npm platform package.

**How to reproduce it (as minimally and precisely as possible)**:
1. mkdir test
2. cd test
3. npm i platform
4. grype this folder.

**Anything else we need to know?**:

**Environment**:
- Output of `grype version`: 0.40.1
- OS (e.g: `cat /etc/os-release` or similar):  linux/arm64

**Comment by willmurphyscode** [comment]
Hi @acthtml thanks for reporting this! I'm not longer able to reproduce this false positive. Here's what I tried:

```
mkdir platform && cd platform && npm init -y && npm install platform && grype dir:.
```

Which prints `No vulnerabilities found`. I believe this was fixed in https://github.com/anchore/grype/releases/tag/v0.60.0, which turns off CPE matching by default for npm packages. I'm closing this because I believe it's resolved, but please let us know if you're still experiencing this issue.

**Comment by willmurphyscode** [comment]
Added `changelog-ignore` because this was fixed in `0.60.0` and so shouldn't be included in the current release's release notes.
